pandas/io/pickle.py

```diff
@@ -182,7 +185,7 @@ def read_pickle(
     2    2    7
     3    3    8
     4    4    9
-    """  # noqa: E501
+    """ 
```

Comment:
I think the only thing left to approve the CI is to remove the trailing white space after `"""`